src/components/application_manager/include/application_manager/core_service.h

```diff
@@ -71,27 +71,25 @@ class CoreService : public Service {
   /**
    * Checks access to requested equipment of vehicle
    * @param app_id id of application
-   * @param function_id name of RPC
+   * @param module type
    * @param params parameters list
    * @param seat seat of owner's mobile device
    * @return return allowed if access exist,
    * manual if need to send question to driver otherwise disallowed
    */
   virtual TypeAccess CheckAccess(const ApplicationId& app_id,
```

Comment:
Why don't we check RPC along with module type? Even if module is allowed we still have to check if RPC belongs to functional groups listed for current app on current device.